plugins/backend/backstage-plugin-argo-cd-backend/src/service/router.ts

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Probably better to leave this notice out for now since other files don't have it either. We'll do a bulk update of these notices at a later stage.

Reply:
addressed.